test_files/abstract/abstract.tsickle.ts

```diff
@@ -8,43 +8,61 @@
  * @abstract
  */
 abstract class Base {
+  abstract simple(): void;
+public abstract publicAbstract(): void;
+public abstract params(x: number[]): void;
+
+  // Verify we properly handle functions without a declared return type.
+  abstract noReturnType();
+
+  // Verify we properly handle functions that expect a return value.
+  abstract hasReturnType(): number;
+/**
+ * @return {void}
+ */
+bar() {
+    this.simple();
+    this.publicAbstract();
+    this.params([]);
+    this.noReturnType();
+    this.hasReturnType();
+  }
+}
+
+function Base_tsickle_Closure_declarations() {
```

Comment:
That's another change. Does `@__PURE__` have an effect on declarations though, as opposed to expressions? I thought it was to mark expressions as no side-effects.